packages/synapse-react-client/src/mocks/team/mockTeam.ts

```diff
@@ -68,6 +74,20 @@ export const mockTeamData4: Readonly<Team> = {
   modifiedBy: String(MOCK_USER_ID_2),
 }
 
+export const mockTeamData5: Readonly<Team> = {
+  id: String(MOCK_TEAM_ID_5),
+  name: 'Mock team with open request',
+  description: 'A team that fake users have already requested to join',
+  icon: '',
+  canPublicJoin: false,
+  canRequestMembership: true,
+  etag: 'f29b79d6-5b63-4641-93c7-30d954b4328c',
+  createdOn: '2013-11-02T01:01:18.373Z',
+  modifiedOn: '2019-01-31T20:34:40.057Z',
+  createdBy: String(MOCK_USER_ID),
+  modifiedBy: String(MOCK_USER_ID_2),
+}
```

Comment:
This new mock team will have existing membership requests. In a later change, it will be used to validate that users cannot submit more than one request to a team